pkgs/by-name/ch/changeme/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+  makeWrapper,
+  installShellFiles,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "changeme";
+  version = "1.2.3";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "ztgrace";
+    repo = "changeme";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-nVHod8xEW05r3ZKfB3OkN2EMGWjfCSA9vOgx+J2OrFI=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    installShellFiles
+  ];
+
+  build-system = with python3Packages; [ setuptools ];
+
+  dependencies = with python3Packages; [
+    responses
+    cerberus
+    mock-ssh-server
+    jinja2
+    logutils
+    lxml
+    netaddr
+    nose
```

Comment:
```suggestion
```
`nose` is a test dependency, and it should not be added to inputs.
see https://github.com/NixOS/nixpkgs/issues/326513